aliasing/api/character.py

```diff
@@ -163,6 +165,49 @@ def create_cc(self, name: str, *args, **kwargs):
             self.delete_cc(name)
         return self.create_cc_nx(name, *args, **kwargs)
 
+    def edit_cc(self, name: str, minVal: str = None, maxVal: str = None, reset: str = None,
+                dispType: str = None, reset_to: str = None, reset_by: str = None,
+                title: str = None, desc: str = None):
```

Comment:
to make the `UNSET` sentinel work, these defaults should be `UNSET` instead of `None`

Reply:
Oh duh.